tests/scss/others/functional_argument_on_same_lines_in_maps.scss

```diff
@@ -0,0 +1,9 @@
+$display-breakpoints: map-deep-merge(
+  (
+    "print-only": "only print",
+    "screen-only": "only screen",
+    "xs-only": "only screen and (max-width: #{map-get($grid-breakpoints, "sm") - 1})",
+    "sm-only": "only screen and (min-width: #{map-get($grid-breakpoints, "sm")}) and (max-width: #{map-get($grid-breakpoints, "md") - 1})",
+  ),
+  $display-breakpoints
+);
```

Comment:
Can we add couple test more? the harder is the better

Reply:
sure, I will try to add. Do you have anything in mind?

Reply:
Something difficult with sass maps/list with interpolation

Reply:
added new testcases